File: EAuctionUI/src/app/Models/ProductBidDetailsModel.ts
import { BidDetailsModel } from "./BidDetailsModel";

export class ProductBidDetailsModel
{
    constructor(){}

    productName: string = ""
    shortDescription: string = ""
    category: string = ""
    startingPrice: number = 0
    bidEndDate: Date = new Date();
    bids: BidDetailsModel[] = []
}

File: EAuction/EAuction.API/BidListingAPI/Models/ProductBidDetailsModel.cs
using System;
using System.Collections.Generic;

namespace EAuction.BidListing.API.Models
{
    public class ProductBidDetailsModel
    {
        public string Id { get; set; }
        public string ProductName { get; set; }
        public string ShortDescription { get; set; }
        public string Category { get; set; }
        public decimal StartingPrice { get; set; }
        public DateTime BidEndDate { get; set; }
        public IList<BidDetailsModel> Bids { get; set; }
    }
}